# option
string INSERT = insert
string DELETE = delete
string UPDATE = update
string SELECT = select

string option
ros_sqlite_database/map data
---
int8 result
ros_sqlite_database/map[] data
